build.gradle.kts

```diff
@@ -81,6 +81,7 @@ kotlin {
         jvmMain {
             dependencies {
                 implementation("com.charleskorn.kaml:kaml:0.72.0")
```

Comment:
Can it be removed?

Reply:
No, it is still used to deserialize the actions file